spec/requests/api/go_cardless/go_cardless_spec.rb

```diff
@@ -6,6 +6,13 @@
   let(:usd_amount) { 9.99 }
   let(:gbp_amount) { 11.55 }
 
+  let(:bad_request_errors) do
+    [{
+      code: 422,
```

Comment:
This appears underneath the heading "We were unable to process your donation!" It's what we display when shit gets fucked up.